gobject-sys/src/lib.rs

```diff
@@ -1589,7 +1590,7 @@ extern "C" {
     pub fn g_object_get_qdata(object: *mut GObject, quark: glib::GQuark) -> gpointer;
     //pub fn g_object_get_valist(object: *mut GObject, first_property_name: *const c_char, var_args: /*Unimplemented*/va_list);
     #[cfg(any(feature = "v2_54", feature = "dox"))]
-    pub fn g_object_getv(object: *mut GObject, n_properties: c_uint, names: *mut c_char, values: GValue);
+    pub fn g_object_getv(object: *mut GObject, n_properties: c_uint, names: *mut *const c_char, values: *mut GValue);
```

Comment:
This changes quite a bit

New gir-files or what is causing this?

Reply:
I have concerns about this change as well.

Reply:
Yes, it was added before in https://github.com/gtk-rs/sys/pull/93 as manual functions.